packages/@uppy/dropbox/types/index.d.ts

```diff
@@ -1,21 +1,17 @@
-import Uppy = require('@uppy/core');
-import CompanionClient = require('@uppy/companion-client');
+import Uppy = require('@uppy/core')
+import CompanionClient = require('@uppy/companion-client')
 
 declare module Dropbox {
-  interface DropboxOptions extends Uppy.PluginOptions, CompanionClient.ProviderOptions {
-    companionUrl: string;
-    companionAllowedHosts: string | RegExp | Array<string | RegExp>;
+  interface DropboxOptions
+    extends Uppy.PluginOptions,
+      CompanionClient.RequestClientOptions {
```

Comment:
why not `CompanionClient.ProviderOptions `? 🤔

Reply:
ProviderOptions includes things like `pluginId: string` which are used by plugins internally to create a `new Provider` instance. i changed the inheritance so there's now a `PublicProviderOptions` interface that only includes the options that you can configure in Plugin options, and `ProviderOptions` inherits from that to add the internal stuff.